sklearn/feature_selection/univariate_selection.py

```diff
@@ -596,7 +596,7 @@ def _get_support_mask(self):
         n_features = len(self.pvalues_)
         sv = np.sort(self.pvalues_)
         selected = sv[sv <= float(self.alpha) / n_features
-                      * np.arange(n_features)]
+                      * (np.arange(n_features) + 1)]
```

Comment:
@jnothman , guido van rossum comments in the bug tracker [on 4/15](http://bugs.python.org/msg263508) that he updated the pep documentation. Interestingly, I thought that W503 [is ignored by default in flake8](https://github.com/PyCQA/flake8/blob/master/src/flake8/defaults.py)